pyannote/audio/cli/train.py

```diff
@@ -77,7 +77,8 @@ def train(cfg: DictConfig) -> Optional[float]:
     fine_tuning = cfg.model["_target_"] == "pyannote.audio.cli.pretrained"
     model = instantiate(cfg.model)
     model.task = task
```

Comment:
In some code I am writing in a private repo, I need to access `task` as soon as `Model.__init__`. So I recommended doing this:

```suggestion
    model = instantiate(cfg.model, task=task)
```